includes/install.php

```diff
@@ -108,9 +108,9 @@ function give_run_install() {
 	$donor_meta = new Give_DB_Donor_Meta();
 	$donor_meta->create_table();
 
-	// Check for PHP Session support, and enable if available.
-	$give_sessions = new Give_Session();
-	$give_sessions->use_php_sessions();
+//	// Check for PHP Session support, and enable if available.
+//	$give_sessions = new Give_Session();
```

Comment:
@ravinderk can you remove this commented code?

Reply:
@DevinWalker I leave this code for reference. If you will approve this pr then I will remove the code

Reply:
Ok sounds good. I'm merging. Looks good and tested well for me.